svg/country-squared/_kosovo.svg

```diff
@@ -56,6 +56,18 @@
        position="600,439.11956"
        orientation="1,0"
        id="guide3688" />
+    <sodipodi:guide
```

Comment:
Out of curiosity, should these `sodipodi` tags be included, or are they an artifact from the tools used?